src/main/scala/mesosphere/marathon/core/matcher/base/util/OfferOperationFactory.scala

```diff
@@ -70,8 +70,9 @@ class OfferOperationFactory(
           .setLabels(reservationLabels.mesosLabels)
           .setPrincipal(principal)
 
+        // TODO @ccaballe volume reservation with the first role
         Mesos.Resource.newBuilder(resource)
-          .setRole(role)
+          .setRole(role(0))
```

Comment:
This is fine for now; let's be sure and document this behavior clearly in a changelog entry describing this change (in this PR, please :) ). You can search the changelog for other deprecation notices we've included in the past.

We can follow up a design session and a change to specify the role that should be used for reservation. This will be a mechanism that will also work for reservation refinements.